api/goldens/aptos_api__tests__transactions_test__test_get_txn_execute_failed_by_missing_script_function_arguments.json

```diff
@@ -6,7 +6,7 @@
   "event_root_hash": "",
   "gas_used": "1",
   "success": false,
-  "vm_status": "Transaction Executed and Committed with Error NUMBER_OF_ARGUMENTS_MISMATCH",
+  "vm_status": "Transaction Executed and Committed with Error FUNCTION_RESOLUTION_FAILURE",
```

Comment:
ditto

Reply:
Nice catch thanks! I missed a couple of TestCoin references. Fixed.